src/commands/TeamsMessaging.ts

```diff
@@ -0,0 +1,73 @@
+import axios from "axios"
+
+export class TeamsMessaging {
+  async main(webhook: string, platformIdent: string, artifactLink: string, isSuccessful: boolean) {
```

Comment:
what does `platformIdent` mean?

Reply:
platform-identifier -> renamed to platform